services/api/src/model/project/measure.rs

```diff
@@ -62,6 +58,10 @@ impl QueryMeasure {
         project_id: ProjectId,
         measure: &MeasureNameId,
     ) -> Result<MeasureId, HttpError> {
+        fn test<T: MeasureDefinition>(measure_str: &str) -> Option<JsonNewMeasure> {
```

Comment:
Maybe make this an auto-implemented associated function for the trait?

Suggested rename `from_name_id`.

Reply:
Done.